scripts/react-tester.js

```diff
@@ -37,52 +53,66 @@ try {
 		console.log( '---------------------------------------------------------------------------' );
 		console.log();
 
-		for ( let key in errorLogs ) {
+		for ( const key in errorLogs ) {
 			console.log( '--- ' + key + ' ---', errorLogs[ key ] );
 		}
 
 		console.log( '--- Some versions failed. See the logs above. ---' );
-		console.log( 'Successful tests:');
+		console.log( 'Successful tests:' );
 		console.log( versionsPassed );
 		console.log();
-		console.log( 'Failed tests:');
+		console.log( 'Failed tests:' );
 		console.log( versionsFailed );
 
 		process.exit( 1 );
 	}
 } catch ( error ) {
-	if ( process.argv[ 2 ] == '-v' ) {
-		console.log( error );
-	} else {
-		console.error( error.stdout );
-	}
-
+	console.log( error );
 	console.log( '--- Unexpected error occured during testing - see the log above. ---' );
-
 	process.exit( 1 );
 }
 
+/**
+ * Gets list of available React versions from npm.
+ * @returns {undefined}
```

Comment:
```suggestion
 * @returns {string[]}
```

?

Reply:
We usually keep one empty line between method description and the rest, like:

```javascript
/**
 * Gets list of available React versions from npm.
 *
 * @returns {undefined}
 */
```

Could you adjust the docs in this file to follow this convention?

Reply:
Okay!